### ISSUE TITLE
Issues with the accuracy of training results.

### ISSUE THREAD

**Issue by BennyTMT** [liaoyuhua/tempo-pytorch#5]
Thank you very much for your implementation!!! but why is the result (Test MAE) I got very large, like 7.42 ? I didn't modify the code, is it because I missed something?

**Comment by BennyTMT** [comment]
<img width="931" alt="Screenshot 2024-03-21 at 8 29 14 PM" src="https://github.com/liaoyuhua/tempo-pytorch/assets/35400831/856c58bc-dcbf-4400-9f59-5554e62be3b0">

**Comment by m6129** [comment]
> Thank you very much for your implementation!!! but why is the result (Test MAE) I got very large, like 7.42 ? I didn't modify the code, is it because I missed something?

please show me all the operations you've performed.

**Comment by mahad852** [comment]
@BennyTMT were you able to resolve this? I am getting similar results, and I am unsure if there is something I missed since the results reported in the paper are much better.

**Comment by BennyTMT** [comment]
@mahad852 unfortunately, not yet. I am waiting the official Repo, but they seem to have no plan to release code yet. You can try email them as well.
